api/http/routing.go

```diff
@@ -222,3 +231,12 @@ func ReleasesRoutes(controller *DeploymentsApiHandlers) []*rest.Route {
 		rest.Get(ApiUrlManagementReleases, controller.GetReleases),
 	}
 }
+
+func FMTConfigURL(hostname, deploymentID, deviceType, deviceID string) string {
+	repl := strings.NewReplacer(
+		":deployment_id", deploymentID,
+		":device_type", deviceType,
+		":device_id", deviceID,
+	)
+	return fmt.Sprintf("https://%s%s", hostname, repl.Replace(ApiUrlDevicesDownloadConfig))
```

Comment:
Isn't Sprintf and over kill here? `"https://"+hostname+repl.Replace(ApiUrlDevicesDownloadConfig)` perhaps?